src/themes.scss

```diff
@@ -76,7 +76,7 @@ $themes: (
     secondary-background: $light-secondary-background,
     primary-color: $primary-color,
     secondary-color: $secondary-color,
-    success-color: $primary-color,
+    success-color: $light-text-color,
```

Comment:
Is this change needed?

Reply:
Well, if `success-color` should be usable at all in the light mode, it should have at least some contrast compared to backgrounds. In the current situation, that was not the case. There were only 2 places where it was actually used...one was those checked words, which didn't work at all. And the second one was the tx amount where the bg is dark (which is really not a common case in light mode). So I would say that this change enables it to be used properly throughout the app.